enhancements/olm/olm-managed-operator-metrics.md

```diff
@@ -22,20 +22,18 @@ status: implementable
 
 ## Summary and Motivation
 
-The primary purpose of this enhancement is to propose a series of changes that would enable [Operator-Lifecycle-Manager (OLM)](https://github.com/operator-framework/operator-lifecycle-manager) managed operators to report metrics to the [OpenShift Monitoring](https://github.com/openshift/cluster-monitoring-operator) [Prometheus Operator](https://github.com/coreos/prometheus-operator).
+The primary purpose of this enhancement is to enable [Operator-Lifecycle-Manager (OLM)](https://github.com/operator-framework/operator-lifecycle-manager) managed operators to easily integrate with the [Prometheus Operator](https://github.com/coreos/prometheus-operator) on OpenShift and Vanilla Kubernetes Clusters. Additionally, this enhancement focuses on enabling OLM managed operators to record metrics with the [OpenShift Monitoring](https://github.com/openshift/cluster-monitoring-operator) Prometheus Operator present on all OpenShift Clusters.
 
-First, OLM will enable operators to interact with a Prometheus Operator deployed on vanilla Kubernetes clusters. This support will be implemented by allowing OLM managed operators to package `ServiceMonitor` and `PrometheusRule` objects within their bundle. OLM will then manage these resources alongside the lifecycle of the operator, as OLM currently does with a number of other resources.
+OLM will support tight integration with the Prometheus Operator by allowing operator authors to package `ServiceMonitor` and `PrometheusRule` objects within their [Operator Bundle](https://github.com/operator-framework/operator-registry/blob/master/docs/design/operator-bundle.md) and will manage these resources alongside the lifecycle of the operator.
 
-Second, OLM will define how an operator author can:
+In an effort to integrate with OpenShift's Monitoring services, OLM will also define how an operator author can surface the following information in the [OpenShift Console UI](https://github.com/openshift/console) when a Cluster Admin is installing an operator:
```

Comment:
nit: the suggested namespace isn't specific to monitoring and will be shown to all users, not only cluster admins. (Although by default only admins can install operators using console OperatorHub.)

Reply:
How would you feel if I were to make the following change:
> OLM will also define how an operator author can surface the following information in the [OpenShift Console UI](https://github.com/openshift/console) when a Cluster Admin is installing an operator, which will be needed when integrating with with OpenShift's Monitoring services:

Reply:
That works for me.

Reply:
Done!